receiver/jaegerreceiver/factory_test.go

```diff
@@ -100,18 +100,6 @@ func TestCreateInvalidHTTPEndpoint(t *testing.T) {
 	assert.Equal(t, 14268, r.(*jReceiver).config.CollectorHTTPPort, "http port should be default")
 }
 
-func TestCreateInvalidTChannelEndpoint(t *testing.T) {
-	factory := Factory{}
-	cfg := factory.CreateDefaultConfig()
-	rCfg := cfg.(*Config)
-
-	rCfg.Protocols[protoThriftTChannel], _ = defaultsForProtocol(protoThriftTChannel)
```

Comment:
One of the tests should be kept, describing what happens when this old option is being used.